index.js

```diff
@@ -25,6 +25,7 @@ function NDRandom(ndc) {
       if (this.ndc[n].x <= this.ndc[n - 1].x) {
         this.err = new Error("Distribution pairs must be increasing");
       }
+      if(this.ndc[n].y>0&&this.ndc[n].y===this.ndc[n - 1].y)this.ndc[n].y*=1.000000000001; // add etta to same numbers
```

Comment:
I'm not familiar with the term etta and Google has been no help here. Admittedly, I got my math degree in 1990 so I'm pretty rusty.

Reply:
I just meant to add a small number
like in 
y =α+βx(t) + ε(t)
the ε letter of statistical error I think it is called etta in latin

to make the number similar to others I just multiplied it by something very small.

it is a like a trick in to add the smallest number to prevent division by errors

it is kind of works in general.